### PR TITLE
Patron: Add communication channel

### PR DESCRIPTION
**Pull request by Garfield-fr**
* NEW: Add communication channel and communication language

How to test:
./scripts/bootstrap
./scripts/setup
Add a new patron user and check fields (communication)

Signed-off-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

### CODE REVIEW

File: rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json [hunk 1]
```diff
@@ -121,6 +121,14 @@
           "patron"
         ]
       }
+    },
+    "communication_channel": {
+      "title": "Communication channel",
+      "type": "string"
```
Comment (jma) [inline rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json]: we can limit the possible values by using an enum: https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values

File: rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json [hunk 2]
```diff
@@ -121,6 +121,14 @@
...
+    },
+    "communication_language": {
+      "title": "Communication language",
+      "type": "string"
```
Comment (jma) [inline rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json]: we can limit the possible values by using an enum: https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values

File: rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json (hunk 1 above)
Comment (Garfield-fr) [reply rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json]: done

File: rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json (hunk 2 above)
Comment (Garfield-fr) [reply rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json]: done

**Pull request merged**